packages/dom/src/animate/animate-style.ts

```diff
@@ -278,3 +291,32 @@ export function animateStyle(
     return animation
   }
 }
+
+function extractKeyframeUnit(keyframe: string | number): string | undefined {
+  if (typeof keyframe !== "string") {
+    return undefined
+  }
+
+  let numberCharFound = false
```

Comment:
why `numberCharFound` never change? What it does?

Reply:
good point, it was a leftover from a refactor. removed!